Makefile

```
@@ -1,25 +1,60 @@
-TERRASPACE     = $(DOCKER) terraspace
-TERRAFORM      = $(DOCKER_COMPOSE) terraform
+DOCKER_COMPOSE    = docker compose
+COMPOSE_ENV_FLAG  = $(if $(wildcard $(ENV_FILE)),--env-file $(ENV_FILE),)
+COMPOSE           = $(DOCKER_COMPOSE) $(COMPOSE_ENV_FLAG)
 
 # Misc
-.DEFAULT_GOAL  = help
-.RECIPEPREFIX  +=
+.DEFAULT_GOAL     = help
+.RECIPEPREFIX    +=
 .PHONY: $(filter-out vendor node_modules,$(MAKECMDGOALS))
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)